lib/utils/interactive.js

```diff
@@ -124,12 +124,19 @@ module.exports = async function(config, outputOptions, processingMessageBuffer)
         if (isExitCtrl(key)) {
             console.clear();
             process.exit();
-        } else if (key.name === 'down') {
-            process.stdout.write(ansiEscapes.cursorNextLine);
-        } else if (key.name === 'up') {
-            process.stdout.write(ansiEscapes.cursorPrevLine);
-        } else if (key.name === 'return') {
-            // TODO: get line and do stuff
+        }
+        switch (key.name) {
+            case 'down':
+                process.stdout.write(ansiEscapes.cursorNextLine);
+                break;
+            case 'up':
+                process.stdout.write(ansiEscapes.cursorPrevLine);
+                break;
+            case 'return':
+                // TODO: get line and do stuff
+                break;
+            default:
+                break;
```

Comment:
It's my personal opinion, I think `if else` provides more flexibility for future need as compared to `switch` and are comparatively more readable too. Consider if you like.

Reply:
I completely agree that `if-else` are easier to read.

> As it turns out, the switch statement is faster in most cases when compared to if-else, but significantly faster only when the number of conditions is large. The primary difference in performance between the two is that the incremental cost of an additional condition is larger for if-else than it is for switch.

https://www.oreilly.com/library/view/high-performance-javascript/9781449382308/ch04.html#:~:targetText=As%20it%20turns%20out%2C%20the,than%20it%20is%20for%20switch%20.

Reply:
@pranshuchittora a valid point. My personal opinions ahead:
That excerpt reminded me of my debate with 1st semester C language teacher why one would use switch over if else and this was the major point for me to get satisfied. Still, If a particular block of code is to be executed only one, I would prefer `if-else` only as the gain is not that much.

Reply:
:smile: That's a good old custom in the engineering of having debates with the profs. Switch case won't match for all conditions if `break` is there.

Reply:
Ofcourse, I meant to say this:
execution of whole switch if we writing switch in a for loop, thats the adequate performance gain.
But if the conditions are meant checked only for a single time, then we need not have a switch.

Reply:
Uhm.. But I think a switch is an optimal & feasible choice.